Mathlib/Topology/MetricSpace/PiNat.lean

```diff
@@ -1079,6 +1080,67 @@ lemma TopologicalSpace.MetrizableSpace.of_countable_separating (f : ∀ i, X →
   (Metric.PiNatEmbed.toPiNatHomeo X Y f continuous_f separating_f).isEmbedding.metrizableSpace
 
 end CompactSpace
+
+open TopologicalSpace Filter unitInterval
+
+variable [MetricSpace X] [SeparableSpace X]
+
+variable (X) in
+/-- Embedding function into 1 direction of countable cube -/
+noncomputable abbrev hCubeEmbedding (n : ℕ) (x : X) : I :=
```

Comment:
This is not a good name for this. We don't have anything called `hCube` in Mathlib. And this *isn't* an embedding (at least, not with this argument order), but you use all of them to build one.
I suggest calling it `distDenseSeq`.

Instead, the docstring should specify exactly what this is:
```suggestion
/-- Given a separable metric space `X`, `denseSeq X : ℕ → X` gives a countable
dense sequence. This measures the distance between `denseSeq X n` and `x`, truncated to the unit interval `I` so that the distances remain bounded.

The function `(fun x n ↦ distDenseSeq n x) : X → ℕ → I` is a mapping from `X` to the Hilbert cube. -/
noncomputable abbrev distDenseSeq (n : ℕ) (x : X) : I :=
```